web-animations-2/Overview.bs

```diff
@@ -1032,6 +1034,7 @@ Add:
 >     Presently start and end delays are zero until such time as percentage
```

Comment:
Might as well swap this out for a markdown paragraph starting with `Note:`, as I included in an earlier suggestion, too. Then you don't have to worry about the `p`'s contents also being improperly indented. ^_^

Reply:
Done.